site/docs/src/json/identity-providers/sonypsn-request.json

```diff
@@ -6,7 +6,7 @@
         "enabled" : true
       }
     },
-    "buttonText" : "Login with Epic Games",
+    "buttonText" : "Login with Sony PlayStation Network",
```

Comment:
Not a big deal, but the default value is `Login with Playstation Network`.

Reply:
Yeah, just saw it was Epic in the sony example and figured I would fix it while I was there. I'll change it.